pulsar/producer.go

```diff
@@ -172,6 +172,15 @@ type ProducerOptions struct {
 
 	// Encryption specifies the fields required to encrypt a message
 	Encryption *ProducerEncryptionInfo
+
+	// EnableChunking controls whether automatic chunking of messages is enabled for the producer. By default, chunking
+	// is disabled.
+	// Chunking can not be enabled when batching is enabled.
+	EnableChunking bool
+
+	// MaxChunkSize is the max size of single chunk payload.
+	// It will actually only take effect if it is smaller than broker.MaxMessageSize
+	MaxChunkSize uint
```

Comment:
It's better to keep it consistent with the java client: `ChunkMaxMessageSize`.

Reply:
I think you r right. I have fix it in the new commit.